test/real_world_cases/seaport_protocol_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule ExWeb3EcRecover.RealWorldCases.SeaportProtocolTest do
```

Comment:
Can we change the name here to full payload test?

Reply:
Intention of this group of modules is to test real world cases. Like 0x (previous real world case) and now Seaport. It tests the lib working on real world test cases. It literally encodes a Seaport OrderComponents.

Reply:
ok [](edited-with-Axolo)